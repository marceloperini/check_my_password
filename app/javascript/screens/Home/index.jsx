import PropTypes from 'prop-types';
import React from 'react';

import Panel from '../../components/Panel';
import PasswordForm from '../../components/PasswordForm';

const Home = () => {
  return (
    <div>
      <Panel>
        <h1>Check My Password</h1>

        <PasswordForm />
      </Panel>
    </div>
  );
};

export default Home;
