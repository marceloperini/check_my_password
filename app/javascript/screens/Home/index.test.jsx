import React from 'react';
import renderer from 'react-test-renderer';

import Home from '.';

describe('Home', () => {
  test('screen', () => {
    const component = renderer.create(
      <Home />
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
