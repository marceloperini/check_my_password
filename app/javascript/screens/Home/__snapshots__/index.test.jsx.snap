// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home screen 1`] = `
<div>
  <div
    className="Panel"
  >
    <h1>
      Check My Password
    </h1>
    <div
      className="PasswordForm"
    >
      <div
        className="TextField"
      >
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="Type the password..."
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
